### DISCUSSION TITLE
Export Kepler Data to PostgreSQL

### DISCUSSION THREAD

**Discussion by AdrianHammond** [sustainable-computing-io/kepler#755] [General]
Has there been any discussions on exporting Kepler data to PostgreSQL database? This was raised in discussion today with organisation who has created a CO2 dashboard which has an SQL backend DB. 

Thanks Adrian

**Comment by rootfs** [comment]
The [promtheus adapter for postgres](https://www.crunchydata.com/blog/using-postgres-to-back-prometheus-for-your-postgresql-monitoring-1) appears to be able to read from kepler exporter and write the metrics to postgres.

**Comment by AdrianHammond** [comment]
closing discussion
